napalm/ios.py

```diff
@@ -520,6 +520,109 @@ def get_interfaces(self):
 
         return interface_list
 
+
+    def get_interfaces_ip(self):
+        '''
+        Get interface ip details
+        Returns a dict of dicts
+
+        Example Output:
+
+        {   u'FastEthernet8': {   'ipv4': {   u'10.66.43.169': {   'prefix-length': 22}}},
```

Comment:
I changed it to underscore

On Fri, Feb 12, 2016 at 10:59 AM, Mircea Ulinic <notifications@github.com>
wrote:

> In napalm/ios.py
> <https://github.com/napalm-automation/napalm/pull/173#discussion_r52767684>
> :
>
> > @@ -520,6 +520,109 @@ def get_interfaces(self):
> >
> >          return interface_list
> >
> > +
> > +    def get_interfaces_ip(self):
> > +        '''
> > +        Get interface ip details
> > +        Returns a dict of dicts
> > +
> > +        Example Output:
> > +
> > +        {   u'FastEthernet8': {   'ipv4': {   u'10.66.43.169': {   'prefix-length': 22}}},
>
> Missed that: prefix-length instead of prefix_length please. And generally
> I think we should avoid "-".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/napalm-automation/napalm/pull/173/files#r52767684>.
>